tide-core/src/error.rs

```diff
@@ -16,7 +16,7 @@ impl std::fmt::Display for StringError {
 #[macro_export]
 macro_rules! err_fmt {
     {$($t:tt)*} => {
-        crate::error::StringError(format!($($t)*))
+        $crate::error::StringError(format!($($t)*))
```

Comment:
Was this intentional?

Reply:
Hi @yoshuawuyts - yes! This is needed since `err_fmt` is in tide_core and also re-exported by `tide`.